src/main/java/Duke.java

```diff
@@ -1,3 +1,5 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
Since logo string is a constant, you can consider making it uppercase as per the module's coding naming standards